src/calculator.js

```diff
@@ -1,48 +1,23 @@
-exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
+module.exports = exports;
+exports._check = () => ({
+  var x;
```

Comment:
Parsing error: Unexpected keyword &#39;var&#39;

###### See http://eslint.org/docs/rules/ for details about this rule